samples/bluetooth/central_smp_client/README.rst

```diff
@@ -1,13 +1,13 @@
 .. _bluetooth_central_dfu_smp:
 
-Bluetooth: Central DFU SMP
-##########################
+Bluetooth: Central SMP Client
+#############################
 
 .. contents::
    :local:
    :depth: 2
 
-The Central DFU SMP sample demonstrates how to use the :ref:`dfu_smp_readme` to connect to an SMP Server and send a simple echo command.
+The Central SMP Client sample demonstrates how to use the :ref:`dfu_smp_readme` to connect to an SMP Server and send a simple echo command.
```

Comment:
I'd really like the SMP abbreviation to be clarified in this readme as SMP is overloaded in Bluetooth.
This could easily be confused with the Security Manager Protocol (SMP) in Bluetoooth (as well as Symmetric Multi-Processing in the zephyr kernel).

So please include "Simple Management Protocol (SMP)" somewhere.

Reply:
Yes, open the abbreviation right on the first occurrence.

Reply:
Done.